<!DOCTYPE html><html lang="zh-CN" data-theme="light"><head><meta charset="UTF-8"><meta http-equiv="X-UA-Compatible" content="IE=edge"><meta name="viewport" content="width=device-width,initial-scale=1"><title>机器学习揭示微生物生态网络中丢失的边及潜在的相互作用机制 | Haizhen's Blog</title><meta name="description" content="一句话评价 数学基础深厚 对涉及的算法进行了适度解释 提供了针对不同类型数据进行特征工程的思路  文章信息 题目：Machine Learning Reveals Missing Edges and Putative Interaction Mechanisms in Microbial Ecosystem Networks 单位：波士顿大学 DOI：10.1128&#x2F;mSystems.00181-"><meta name="keywords" content="random forest,network,coculture"><meta name="author" content="Zhu Haizhen,zhuhaizhen1024@163.com"><meta name="copyright" content="Zhu Haizhen"><meta name="format-detection" content="telephone=no"><link rel="shortcut icon" href="/image/favicon.png"><link rel="canonical" href="https://zhuhaizhen.github.io/2020/11/21/1121/"><meta http-equiv="Cache-Control" content="no-transform"><meta http-equiv="Cache-Control" content="no-siteapp"><link rel="preconnect" href="//cdn.jsdelivr.net"/><link rel="preconnect" href="//hm.baidu.com"/><link rel="preconnect" href="//fonts.googleapis.com" crossorigin="crossorigin"/><link rel="preconnect" href="//busuanzi.ibruce.info"/><meta property="og:type" content="article"><meta property="og:title" content="机器学习揭示微生物生态网络中丢失的边及潜在的相互作用机制"><meta property="og:url" content="https://zhuhaizhen.github.io/2020/11/21/1121/"><meta property="og:site_name" content="Haizhen's Blog"><meta property="og:description" content="一句话评价 数学基础深厚 对涉及的算法进行了适度解释 提供了针对不同类型数据进行特征工程的思路  文章信息 题目：Machine Learning Reveals Missing Edges and Putative Interaction Mechanisms in Microbial Ecosystem Networks 单位：波士顿大学 DOI：10.1128&#x2F;mSystems.00181-"><meta property="og:image" content="https://gitee.com/serene_site/image/tree/master/img/nature-5296373_1920.jpg"><meta property="article:published_time" content="2020-11-21T03:43:29.000Z"><meta property="article:modified_time" content="2022-03-31T12:42:10.045Z"><meta name="twitter:card" content="summary"><script>var activateDarkMode = function () {
  document.documentElement.setAttribute('data-theme', 'dark')
  if (document.querySelector('meta[name="theme-color"]') !== null) {
    document.querySelector('meta[name="theme-color"]').setAttribute('content', '#000')
  }
}
var activateLightMode = function () {
  document.documentElement.setAttribute('data-theme', 'light')
  if (document.querySelector('meta[name="theme-color"]') !== null) {
    document.querySelector('meta[name="theme-color"]').setAttribute('content', '#fff')
  }
}

var getCookies = function (name) {
  const value = `; ${document.cookie}`
  const parts = value.split(`; ${name}=`)
  if (parts.length === 2) return parts.pop().split(';').shift()
}

var autoChangeMode = 'false'
var t = getCookies('theme')
if (autoChangeMode === '1') {
  var isDarkMode = window.matchMedia('(prefers-color-scheme: dark)').matches
  var isLightMode = window.matchMedia('(prefers-color-scheme: light)').matches
  var isNotSpecified = window.matchMedia('(prefers-color-scheme: no-preference)').matches
  var hasNoSupport = !isDarkMode && !isLightMode && !isNotSpecified

  if (t === undefined) {
    if (isLightMode) activateLightMode()
    else if (isDarkMode) activateDarkMode()
    else if (isNotSpecified || hasNoSupport) {
      console.log('You specified no preference for a color scheme or your browser does not support it. I Schedule dark mode during night time.')
      var now = new Date()
      var hour = now.getHours()
      var isNight = hour <= 6 || hour >= 18
      isNight ? activateDarkMode() : activateLightMode()
    }
    window.matchMedia('(prefers-color-scheme: dark)').addListener(function (e) {
      if (Cookies.get('theme') === undefined) {
        e.matches ? activateDarkMode() : activateLightMode()
      }
    })
  } else if (t === 'light') activateLightMode()
  else activateDarkMode()
} else if (autoChangeMode === '2') {
  now = new Date()
  hour = now.getHours()
  isNight = hour <= 6 || hour >= 18
  if (t === undefined) isNight ? activateDarkMode() : activateLightMode()
  else if (t === 'light') activateLightMode()
  else activateDarkMode()
} else {
  if (t === 'dark') activateDarkMode()
  else if (t === 'light') activateLightMode()
}</script><link rel="stylesheet" href="/css/index.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fortawesome/fontawesome-free/css/all.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/@fancyapps/fancybox@latest/dist/jquery.fancybox.min.css"><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/node-snackbar/dist/snackbar.min.css"><link rel="prev" title="SVM算法" href="https://zhuhaizhen.github.io/2020/11/29/SVM/"><link rel="next" title="基因组学深度学习入门" href="https://zhuhaizhen.github.io/2020/11/14/1114/"><link rel="stylesheet" type="text/css" href="https://cdn.jsdelivr.net/npm/instantsearch.js@2.10.5/dist/instantsearch.min.css"><script src="https://cdn.jsdelivr.net/npm/instantsearch.js@2.10.5/dist/instantsearch.min.js" defer></script><script>var _hmt = _hmt || [];
(function() {
  var hm = document.createElement("script");
  hm.src = "https://hm.baidu.com/hm.js?6b1c40d9cb0aeba42b229c550f73967b";
  var s = document.getElementsByTagName("script")[0]; 
  s.parentNode.insertBefore(hm, s);
})();
</script><link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Titillium+Web&amp;display=swap"><script>var GLOBAL_CONFIG = { 
  root: '/',
  algolia: {"appId":"ULAMU2VKLC","apiKey":"074084189662d00e3b579aef488f16a5","indexName":"hexo-blog","hits":{"per_page":6},"languages":{"input_placeholder":"搜索文章","hits_empty":"找不到您查询的内容:${query}","hits_stats":"找到 ${hits} 条结果，用时 ${time} 毫秒"}},
  localSearch: undefined,
  translate: undefined,
  copy: {
    success: '复制成功',
    error: '复制错误',
    noSupport: '浏览器不支持'
  },
  bookmark: {
    message_prev: '按',
    message_next: '键将本页加入书签'
  },
  runtime_unit: '天',
  runtime: true,
  copyright: undefined,
  ClickShowText: undefined,
  medium_zoom: false,
  fancybox: true,
  Snackbar: {"bookmark":{"message_prev":"按","message_next":"键将本页加入书签"},"chs_to_cht":"你已切换为繁体","cht_to_chs":"你已切换为简体","day_to_night":"你已切换为深色模式","night_to_day":"你已切换为浅色模式","bgLight":"#49b1f5","bgDark":"#121212","position":"bottom-left"},
  justifiedGallery: {
    js: 'https://cdn.jsdelivr.net/npm/justifiedGallery/dist/js/jquery.justifiedGallery.min.js',
    css: 'https://cdn.jsdelivr.net/npm/justifiedGallery/dist/css/justifiedGallery.min.css'
  },
  baiduPush: false,
  highlightCopy: true,
  highlightLang: true,
  isPhotoFigcaption: false,
  islazyload: true,
  isanchor: false    
}</script><script>var GLOBAL_CONFIG_SITE = { 
  isPost: true,
  isHome: false,
  isHighlightShrink: false,
  isSidebar: true
  }</script><noscript><style>
#nav {
  opacity: 1
}
.justified-gallery img{
  opacity: 1
}
</style></noscript><meta name="generator" content="Hexo 4.2.1"></head><body><div id="loading-box"><div class="loading-left-bg"></div><div class="loading-right-bg"></div><div class="spinner-box"><div class="configure-border-1"><div class="configure-core"></div></div><div class="configure-border-2"><div class="configure-core"></div></div><div class="loading-word">加载中...</div></div></div><div id="mobile-sidebar"><div id="menu_mask"></div><div id="mobile-sidebar-menus"><div class="mobile_author_icon"><img class="avatar-img" src="/image/avatar.jpg" onerror="onerror=null;src='/img/friend_404.gif'" alt="avatar"/></div><div class="mobile_post_data"><div class="mobile_data_item is-center"><div class="mobile_data_link"><a href="/archives/"><div class="headline">文章</div><div class="length_num">23</div></a></div></div><div class="mobile_data_item is-center">      <div class="mobile_data_link"><a href="/tags/"><div class="headline">标签</div><div class="length_num">32</div></a></div></div><div class="mobile_data_item is-center">     <div class="mobile_data_link"><a href="/categories/"><div class="headline">分类</div><div class="length_num">5</div></a></div></div></div><hr/><div class="menus_items"><div class="menus_item"><a class="site-page" href="/"><i class="fa-fw fas fa-home"></i><span> 首页</span></a></div><div class="menus_item"><a class="site-page" href="/archives/"><i class="fa-fw fas fa-archive"></i><span> 归档</span></a></div><div class="menus_item"><a class="site-page" href="/tags/"><i class="fa-fw fas fa-tags"></i><span> 标签</span></a></div><div class="menus_item"><a class="site-page" href="/categories/"><i class="fa-fw fas fa-folder-open"></i><span> 分类</span></a></div><div class="menus_item"><a class="site-page" href="/about/"><i class="fa-fw fas fa-heart"></i><span> 关于</span></a></div><div class="menus_item"><a class="site-page" href="/collection/"><i class="fa-fw fas fa-paw"></i><span> 随手记</span></a></div><div class="menus_item"><a class="site-page" href="/messageboard/"><i class="fa-fw fas fa-comments"></i><span> 留言板</span></a></div></div></div></div><i class="fas fa-arrow-right on" id="toggle-sidebar"></i><div id="sidebar"><div class="sidebar-toc"><div class="sidebar-toc__title">目录</div><div class="sidebar-toc__progress"><span class="progress-notice">你已经读了</span><span class="progress-num">0</span><span class="progress-percentage">%</span><div class="sidebar-toc__progress-bar">     </div></div><div class="sidebar-toc__content"><ol class="toc"><li class="toc-item toc-level-2"><a class="toc-link" href="#一句话评价"><span class="toc-number">1.</span> <span class="toc-text">一句话评价</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#文章信息"><span class="toc-number">2.</span> <span class="toc-text">文章信息</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#要解决的问题"><span class="toc-number">3.</span> <span class="toc-text">要解决的问题</span></a></li><li class="toc-item toc-level-2"><a class="toc-link" href="#实验设计"><span class="toc-number">4.</span> <span class="toc-text">实验设计</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#特征工程"><span class="toc-number">4.1.</span> <span class="toc-text">特征工程</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#肠道微生物相互作用数据"><span class="toc-number">4.2.</span> <span class="toc-text">肠道微生物相互作用数据</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#营养缺陷型大肠杆菌数据"><span class="toc-number">4.3.</span> <span class="toc-text">营养缺陷型大肠杆菌数据</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#土壤群落数据"><span class="toc-number">4.4.</span> <span class="toc-text">土壤群落数据</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#随机森林建模"><span class="toc-number">4.5.</span> <span class="toc-text">随机森林建模</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#数据及代码"><span class="toc-number">4.6.</span> <span class="toc-text">数据及代码</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#结果"><span class="toc-number">5.</span> <span class="toc-text">结果</span></a><ol class="toc-child"><li class="toc-item toc-level-3"><a class="toc-link" href="#表征两两相互作用"><span class="toc-number">5.1.</span> <span class="toc-text">表征两两相互作用</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#在计算机预测的人肠道微生物相互作用数据中的应用"><span class="toc-number">5.2.</span> <span class="toc-text">在计算机预测的人肠道微生物相互作用数据中的应用</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#在营养缺陷型大肠杆菌数据中的应用"><span class="toc-number">5.3.</span> <span class="toc-text">在营养缺陷型大肠杆菌数据中的应用</span></a></li><li class="toc-item toc-level-3"><a class="toc-link" href="#在土壤细菌群落数据中的应用"><span class="toc-number">5.4.</span> <span class="toc-text">在土壤细菌群落数据中的应用</span></a></li></ol></li><li class="toc-item toc-level-2"><a class="toc-link" href="#参考文献"><span class="toc-number">6.</span> <span class="toc-text">参考文献</span></a></li></ol></div></div></div><div id="body-wrap"><header class="post-bg" id="page-header" style="background-image: url(https://gitee.com/serene_site/image/tree/master/img/nature-5296373_1920.jpg)"><nav id="nav"><span class="pull-left" id="blog_name"><a class="blog_title" id="site-name" href="/">Haizhen's Blog</a></span><span class="pull-right menus"><div id="search_button"><a class="site-page social-icon search"><i class="fas fa-search fa-fw"></i><span> 搜索</span></a></div><div class="menus_items"><div class="menus_item"><a class="site-page" href="/"><i class="fa-fw fas fa-home"></i><span> 首页</span></a></div><div class="menus_item"><a class="site-page" href="/archives/"><i class="fa-fw fas fa-archive"></i><span> 归档</span></a></div><div class="menus_item"><a class="site-page" href="/tags/"><i class="fa-fw fas fa-tags"></i><span> 标签</span></a></div><div class="menus_item"><a class="site-page" href="/categories/"><i class="fa-fw fas fa-folder-open"></i><span> 分类</span></a></div><div class="menus_item"><a class="site-page" href="/about/"><i class="fa-fw fas fa-heart"></i><span> 关于</span></a></div><div class="menus_item"><a class="site-page" href="/collection/"><i class="fa-fw fas fa-paw"></i><span> 随手记</span></a></div><div class="menus_item"><a class="site-page" href="/messageboard/"><i class="fa-fw fas fa-comments"></i><span> 留言板</span></a></div></div><span class="toggle-menu close"><a class="site-page"><i class="fas fa-bars fa-fw"></i></a></span></span></nav><div id="post-info"><div id="post-title"><div class="posttitle">机器学习揭示微生物生态网络中丢失的边及潜在的相互作用机制</div></div><div id="post-meta"><div class="meta-firstline"><time class="post-meta__date"><span class="post-meta__date-created" title="发表于 2020-11-21 11:43:29"><i class="far fa-calendar-alt fa-fw"></i> 发表于 2020-11-21</span><span class="post-meta__separator">|</span><span class="post-meta__date-updated" title="更新于 2022-03-31 20:42:10"><i class="fas fa-history fa-fw"></i> 更新于 2022-03-31</span></time><span class="post-meta__categories"><span class="post-meta__separator">|</span><i class="fas fa-inbox fa-fw post-meta__icon"></i><a class="post-meta__categories" href="/categories/%E6%96%87%E7%8C%AE%E9%98%85%E8%AF%BB/">文献阅读</a></span></div><div class="meta-secondline"> <span class="post-meta-wordcount"><i class="far fa-file-word fa-fw post-meta__icon"></i><span>字数总计:</span><span class="word-count">2.1k</span><span class="post-meta__separator">|</span><i class="far fa-clock fa-fw post-meta__icon"></i><span>阅读时长: 7 分钟</span></span></div><div class="meta-thirdline"><span class="post-meta-pv-cv"><span class="post-meta__separator">|</span><i class="far fa-eye fa-fw post-meta__icon"></i><span>阅读量:</span><span id="busuanzi_value_page_pv"></span></span><span class="post-meta-commentcount"><span class="post-meta__separator">|</span><i class="far fa-comments fa-fw post-meta__icon"></i><span>评论数:</span><a href="/2020/11/21/1121/#post-comment" itemprop="discussionUrl"><span class="valine-comment-count comment-count" data-xid="/2020/11/21/1121/" itemprop="commentCount"></span></a></span></div></div></div></header><main class="layout_post" id="content-inner"><article id="post"><div class="post-content" id="article-container"><h2 id="一句话评价"><a href="#一句话评价" class="headerlink" title="一句话评价"></a>一句话评价</h2><ul>
<li>数学基础深厚</li>
<li>对涉及的算法进行了适度解释</li>
<li>提供了针对不同类型数据进行特征工程的思路</li>
</ul>
<h2 id="文章信息"><a href="#文章信息" class="headerlink" title="文章信息"></a>文章信息</h2><ul>
<li><strong>题目</strong>：Machine Learning Reveals Missing Edges and Putative Interaction Mechanisms in Microbial Ecosystem Networks</li>
<li><strong>单位</strong>：波士顿大学</li>
<li><strong>DOI</strong>：10.1128/mSystems.00181-18</li>
<li><strong>杂志</strong>：mSystems (IF 6.633)</li>
<li><strong>分类</strong>：Research article</li>
</ul>
<h2 id="要解决的问题"><a href="#要解决的问题" class="headerlink" title="要解决的问题"></a>要解决的问题</h2><p>提出了用数学方法表征微生物相互作用的概念框架，并应用于三个实例：</p>
<blockquote>
<p>三个实例都有菌株两两共培养的实验数据</p>
</blockquote>
<ol>
<li>100个人肠道细菌代谢模型的模拟群落</li>
<li>包含14个菌株的大肠杆菌氨基酸营养缺陷型群落</li>
<li>分离自同一土壤样品的20个微生物组成的群落</li>
</ol>
<h2 id="实验设计"><a href="#实验设计" class="headerlink" title="实验设计"></a>实验设计</h2><h3 id="特征工程"><a href="#特征工程" class="headerlink" title="特征工程"></a>特征工程</h3><ul>
<li><p>对于给定群落<code>C</code>，菌株<code>i</code>在菌株<code>j</code>存在时的响应记为群落矩阵 $X$ 中的 $X_{ij}$ 元素</p>
</li>
<li><p>$X_{ij}$ 代表共培养结束时菌株<code>i</code>的标准化丰度或者与菌株<code>j</code>共培养是否能够存活的0-1数据</p>
</li>
<li><p>对<code>C</code>中的每一个菌株<code>i</code>定义一个包含<code>n</code>个特征的特征向量 $F^{(i)}$，包含是否存在特定基因、代谢功能等</p>
</li>
<li><p>$F_k^{(i)}$ 根据特征<code>k</code>是否存在于菌株<code>i</code>记为1或0</p>
</li>
<li><p>每一对菌株(i, j)关联一个串联特征向量 $F^{(i,j)} = [F^{(i)},F^{(j)}]$</p>
</li>
<li><p>一般情况下，<strong>$F^{(i,j)} \neq F^{(j,i)}$ 且 $X_{ij} \neq X_{ji}$</strong></p>
</li>
</ul>
<h3 id="肠道微生物相互作用数据"><a href="#肠道微生物相互作用数据" class="headerlink" title="肠道微生物相互作用数据"></a>肠道微生物相互作用数据</h3><ul>
<li><p>数据来自已发表的文章<a href="#参考文献">[1]</a></p>
</li>
<li><p>流平衡分析 (flux balance analysis, FBA)，是一种基于约束的稳态模型<a href="#参考文献">[2,3,4,5]</a></p>
<ul>
<li><p>根据基因组注释构建代谢网络</p>
</li>
<li><p>利用代谢网络构建化学计量矩阵<code>S</code></p>
<blockquote>
<p>$S_{ij}$ 元素表示通过反应<code>j</code>消耗或产生了几分子的<code>i</code></p>
</blockquote>
</li>
<li><p>基于稳态假设，根据边界约束条件定义环境中营养物质的可获得性</p>
<blockquote>
<p>约束来自两个方面：能量守恒等物理规律及底物的供应等客观限制。这个步骤是确定约束条件和目标函数的过程</p>
</blockquote>
</li>
<li><p>通过线性优化获得使生物量最大的流分布</p>
</li>
</ul>
</li>
<li><p>动态流平衡分析 (dynamic flux balance analysis, dFBA)：仍然假设细胞内的代谢物处于稳态，但环境中总的生物量和代谢物视作时间依赖的变量</p>
<ul>
<li><p>使用之前已发表的<code>COMETS</code>平台进行仿真<a href="#参考文献">[6]</a></p>
</li>
<li><p>选择了100个代谢模型 (菌株)</p>
</li>
<li><p>设置所有菌株单独培养都能生长的培养基</p>
</li>
<li><p>两两共培养仿真设置为混合良好的分批培养 (batch culture)</p>
</li>
<li><p>如果 $B_{ij}$ 是菌株<code>i</code>在与菌株<code>j</code>共培养时最终的生物量，$B_{ii}$ 是菌株<code>i</code>单独培养时最终的生物量，定义 $X_{ij} = \frac {B_{ij} - B_{ii}} {B_{ii}}$</p>
<blockquote>
<p>如果$X_{ij} &lt; 0$，表明菌株<code>i</code>受菌株<code>j</code>抑制；如果$X_{ij} = 0$，表明菌株<code>j</code>对菌株<code>i</code>没有影响</p>
</blockquote>
</li>
</ul>
</li>
<li><p>本例中特征向量 $F^{(i)}$ 定义为物种<code>i</code>的基因组中是否存在 (1 / 0)化学计量矩阵<code>S</code>中的194个可能的反应</p>
</li>
<li><p>基于两个特征向量 $F^{(i)}$ 和 $F^{(j)}$ 之间的<code>Jaccard</code>距离的分类器用于与随机森林模型进行比较，定义为：<br>$$ JD(F^{(i)},F^{(j)}) = 1 - \frac {F^{(i)} \bigcap F^{(j)}} {F^{(i)} \bigcup F^{(j)}} $$</p>
</li>
</ul>
<h3 id="营养缺陷型大肠杆菌数据"><a href="#营养缺陷型大肠杆菌数据" class="headerlink" title="营养缺陷型大肠杆菌数据"></a>营养缺陷型大肠杆菌数据</h3><ul>
<li><p>数据来自已发表的文章<a href="#参考文献">[7]</a></p>
</li>
<li><p>通过基因敲除获得14个氨基酸营养缺陷型大肠杆菌菌株</p>
</li>
<li><p>共培养使用M9葡萄糖培养基200 μl体系，30 ℃培养84 h</p>
</li>
<li><p>通过菌落计数、qPCR确定菌株比例</p>
</li>
<li><p>本例中特征向量 $F^{(i)}$ 定义为是否能合成14种氨基酸，共培养表型 $X_{ij}$ 定义为菌株<code>i</code>在与菌株<code>j</code>共培养时，最终生物量变化的倍数</p>
</li>
</ul>
<h3 id="土壤群落数据"><a href="#土壤群落数据" class="headerlink" title="土壤群落数据"></a>土壤群落数据</h3><ul>
<li><p>数据来自已发表的文章<a href="#参考文献">[8]</a></p>
</li>
<li><p>菌株的存活通过5轮稀释后进行菌落计数及高通量测序获得</p>
</li>
<li><p>为构建特征向量 $F^{(i)}$，从GenBank下载各菌株的16S rRNA序列，用PICRUSt预测其KEGG模块。其中18个菌株通过预测获得了KEGG模块，将特征向量定义为是否存在79个KEGG模块</p>
</li>
<li><p>$X_{ij}$ 定义为菌株<code>i</code>与菌株<code>j</code>共培养时能否存活</p>
</li>
</ul>
<h3 id="随机森林建模"><a href="#随机森林建模" class="headerlink" title="随机森林建模"></a>随机森林建模</h3><ul>
<li><p>使用 <code>RandomForest</code> R包</p>
</li>
<li><p>使用默认参数</p>
</li>
<li><p>根据训练集的<code>OOB</code>及5-倍交叉验证评估准确率，绘制<code>ROC</code>曲线</p>
</li>
<li><p>使用 <code>forestFloor</code> R包计算特征对分类的贡献</p>
</li>
</ul>
<h3 id="数据及代码"><a href="#数据及代码" class="headerlink" title="数据及代码"></a>数据及代码</h3><p><strong><a href="https://github.com/ddimucci/MicrobialCommunities" target="_blank" rel="noopener">https://github.com/ddimucci/MicrobialCommunities</a></strong></p>
<h2 id="结果"><a href="#结果" class="headerlink" title="结果"></a>结果</h2><h3 id="表征两两相互作用"><a href="#表征两两相互作用" class="headerlink" title="表征两两相互作用"></a>表征两两相互作用</h3><ul>
<li><p>特征向量可以通过串联各种生物学相关特征进行构建，如：是否编码特定基因或代谢功能、系统发育地位、发现该微生物的环境等</p>
</li>
<li><p>这些特征向量结合已知的相互作用关系可以用来训练机器学习模型，进而预测未观察到的相互作用</p>
</li>
</ul>
<p><img src= "/img/loading.gif" data-src="https://gitee.com/serene_site/image/tree/master/img/20201121162034.png" alt="图1"></p>
<h3 id="在计算机预测的人肠道微生物相互作用数据中的应用"><a href="#在计算机预测的人肠道微生物相互作用数据中的应用" class="headerlink" title="在计算机预测的人肠道微生物相互作用数据中的应用"></a>在计算机预测的人肠道微生物相互作用数据中的应用</h3><ul>
<li><p>特征向量为每个菌株是否编码各种营养物质交换反应的基因 (图2A)</p>
</li>
<li><p>仿真实验表型为菌株两两共培养最终的生物量相对变化 (图2B)</p>
<blockquote>
<p>5,563个样本的最终相对生物量减少，3,917个样本的最终相对生物量基本不变，420个样本的最终相对生物量增加</p>
</blockquote>
</li>
<li><p>对整个数据集进行训练，准确率约为90.5%。ROC曲线表明随机森林模型比单纯根据 <code>Jaccard</code> 距离预测负相互作用效果更好 (图2C)</p>
</li>
<li><p>通过绘制学习曲线判断群落大小及训练数据的多少对准确率的影响：群落越大，达到较高准确率需要的训练数据(实验结果)越少 (图2D)</p>
</li>
<li><p>获得了最重要的20个特征 (图2E)</p>
<blockquote>
<p>纵坐标的p上标表示该特征是属于共培养对象的</p>
</blockquote>
</li>
</ul>
<p><img src= "/img/loading.gif" data-src="https://gitee.com/serene_site/image/tree/master/img/20201121162845.png" alt="图2"></p>
<ul>
<li>随机森林模型还可定量计算特定变量对单个样品的分类有多大影响，从而用于推测相互作用的机制 (图3)</li>
</ul>
<p><img src= "/img/loading.gif" data-src="https://gitee.com/serene_site/image/tree/master/img/20201121171111.png" alt="图3"></p>
<h3 id="在营养缺陷型大肠杆菌数据中的应用"><a href="#在营养缺陷型大肠杆菌数据中的应用" class="headerlink" title="在营养缺陷型大肠杆菌数据中的应用"></a>在营养缺陷型大肠杆菌数据中的应用</h3><ul>
<li><p>任意两个菌株的相互作用依赖于直接的缺陷氨基酸交换 (图4A)</p>
</li>
<li><p>利用共培养的最终生长倍数变化构建表型矩阵，以变化2倍作为“强相互作用”和“弱相互作用”的阈值 (图4B)</p>
</li>
<li><p>随机森林模型的准确率约为79.2%，根据ROC曲线，表现比基于生物合成代价矩阵的预测更好 (图4C)</p>
</li>
<li><p>变量重要性分析表明，缺陷的氨基酸对于分类的准确率影响更大 (图4E)</p>
</li>
</ul>
<p><img src= "/img/loading.gif" data-src="https://gitee.com/serene_site/image/tree/master/img/20201121171923.png" alt="图4"></p>
<ul>
<li>对特征贡献的分析发现，缺失某种氨基酸对随机森林分类器的影响比存在某种氨基酸更大</li>
</ul>
<h3 id="在土壤细菌群落数据中的应用"><a href="#在土壤细菌群落数据中的应用" class="headerlink" title="在土壤细菌群落数据中的应用"></a>在土壤细菌群落数据中的应用</h3><ul>
<li><p>随机森林模型准确率约为79.4%</p>
</li>
<li><p>ROC曲线表明随机森林模型表现比根据初始生长速率的预测效果更好 (图5A)</p>
</li>
<li><p>学习曲线趋势与基于仿真数据的趋势一致 (图5B)</p>
</li>
</ul>
<p><img src= "/img/loading.gif" data-src="https://gitee.com/serene_site/image/tree/master/img/20201121174846.png" alt="图5"></p>
<h2 id="参考文献"><a href="#参考文献" class="headerlink" title="参考文献"></a>参考文献</h2><p>[1] Bauer E, Laczny CC, Magnusdottir S, Wilmes P, Thiele I. 2015. Phenotypic differentiation of gastrointestinal microbes is reflected in their encoded metabolic repertoires. Microbiome 3:55.<br>[2] Maarleveld TR, Khandelwal RA, Olivier BG, Teusink B, Bruggeman FJ. 2013. Basic concepts and principles of stoichiometric modeling of metabolic networks. Biotechnol J 8:997–1008.<br>[3] Raman K, Chandra N. 2009. Flux balance analysis of biological systems: applications and challenges. Brief Bioinform 10:435– 449.<br>[4] O’Brien EJ, Monk JM, Palsson BO. 2015. Using genome-scale models to predict biological capabilities. Cell 161:971–987.<br>[5] Orth JD, Thiele I, Palsson BØ. 2010. What is flux balance analysis? Nat Biotechnol 28:245–248.<br>[6] Harcombe WR, Riehl WJ, Dukovski I, Granger BR, Betts A, Lang AH, Bonilla G, Kar A, Leiby N, Mehta P, Marx CJ, Segrè D. 2014. Metabolic resource allocation in individual microbes determines ecosystem interactions and spatial dynamics. Cell Rep 7:1104–1115.<br>[7] Mee MT, Collins JJ, Church GM, Wang HH. 2014. Syntrophic exchange in synthetic microbial communities. Proc Natl Acad Sci U S A 111: E2149–E2156.<br>[8] Higgins LM, Friedman J, Shen H, Gore J. 2017. Co-occurring soil bacteria exhibit a robust competitive hierarchy and lack of non-transitive interactions. bioRxiv    </p>
</div><div class="post-copyright"><div class="post-copyright__author"><span class="post-copyright-meta">文章作者: </span><span class="post-copyright-info"><a href="mailto:zhuhaizhen1024@163.com">Zhu Haizhen</a></span></div><div class="post-copyright__type"><span class="post-copyright-meta">文章链接: </span><span class="post-copyright-info"><a href="https://zhuhaizhen.github.io/2020/11/21/1121/">https://zhuhaizhen.github.io/2020/11/21/1121/</a></span></div><div class="post-copyright__notice"><span class="post-copyright-meta">版权声明: </span><span class="post-copyright-info">本博客所有文章除特别声明外，均采用 <a href="https://creativecommons.org/licenses/by-nc-sa/4.0/" target="_blank">CC BY-NC-SA 4.0</a> 许可协议。转载请注明来自 <a href="https://zhuhaizhen.github.io" target="_blank">Haizhen's Blog</a>！</span></div></div><div class="tag_share"><div class="post-meta__tag-list"><a class="post-meta__tags" href="/tags/random-forest/">random forest</a><a class="post-meta__tags" href="/tags/network/">network</a><a class="post-meta__tags" href="/tags/coculture/">coculture</a></div><div class="post_share"><div class="social-share" data-image="https://gitee.com/serene_site/image/tree/master/img/sky-5347490_1920.jpg" data-sites="facebook,twitter,wechat,weibo,qq"></div><link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/social-share.js/dist/css/share.min.css"/><script src="https://cdn.jsdelivr.net/npm/social-share.js/dist/js/social-share.min.js"></script></div></div><div class="post-reward"><button class="reward-button"><i class="fas fa-qrcode"></i> 打赏<div class="reward-main"><ul class="reward-all"><li class="reward-item"><img class="post-qr-code__img" src="/image/wechat.jpg" alt="微信" onclick="window.open('/image/wechat.jpg')"/><div class="post-qr-code__desc">微信</div></li><li class="reward-item"><img class="post-qr-code__img" src="/image/alipay.jpg" alt="支付宝" onclick="window.open('/image/alipay.jpg')"/><div class="post-qr-code__desc">支付宝</div></li></ul></div></button></div><nav class="pagination-post" id="pagination"><div class="prev-post pull-left"><a href="/2020/11/29/SVM/"><img class="prev-cover" data-src="https://gitee.com/serene_site/image/tree/master/img/mountains-1412683_1280.png" onerror="onerror=null;src='/img/404.jpg'"><div class="pagination-info"><div class="label">上一篇</div><div class="prev_info">SVM算法</div></div></a></div><div class="next-post pull-right"><a href="/2020/11/14/1114/"><img class="next-cover" data-src="https://gitee.com/serene_site/image/tree/master/img/nature-5296373_1920.jpg" onerror="onerror=null;src='/img/404.jpg'"><div class="pagination-info"><div class="label">下一篇</div><div class="next_info">基因组学深度学习入门</div></div></a></div></nav><div class="relatedPosts"><div class="relatedPosts_headline"><i class="fas fa-thumbs-up fa-fw"></i><span> 相关推荐</span></div><div class="relatedPosts_list"><div class="relatedPosts_item"><a href="/2020/11/07/1107/" title="机器学习辅助分析上千个基因组草图揭示活性污泥工艺中的具体特征"><img class="relatedPosts_cover" data-src="https://gitee.com/serene_site/image/tree/master/img/book-5280551_1920.jpg"><div class="relatedPosts_main is-center"><div class="relatedPosts_date"><i class="far fa-calendar-alt fa-fw"></i> 2020-11-07</div><div class="relatedPosts_title">机器学习辅助分析上千个基因组草图揭示活性污泥工艺中的具体特征</div></div></a></div></div></div><hr><div id="post-comment"><div class="comment_headling"><i class="fas fa-comments fa-fw"></i><span> 评论</span></div><div class="vcomment" id="vcomment"></div><script src="https://cdn.jsdelivr.net/npm/valine/dist/Valine.min.js"></script><script>var requestSetting = function (from,set) {
  var from = from
  var setting = set.split(',').filter(function(item){
  return from.indexOf(item) > -1
  });
  setting = setting.length == 0 ? from :setting;
  return setting
}

var guestInfo = requestSetting(['nick','mail','link'],'昵称,邮箱,网址')
var requiredFields = requestSetting(['nick','mail'],'昵称,邮箱')

window.valine = new Valine({
  el:'#vcomment',
  appId: 'yDcQjFWTIoApJ6qYc7DCVIjO-gzGzoHsz',
  appKey: 'y66ni3n3AtYNrLgaqdJGzAPo',
  placeholder: '请留下你的昵称和邮箱，方便接收回复',
  avatar: 'monsterid',
  meta: guestInfo,
  pageSize: '10',
  lang: 'zh-CN',
  recordIP: false,
  serverURLs: '',
  emojiCDN: '',
  emojiMaps: "",
  enableQQ: false,
  requiredFields: requiredFields
});</script></div></article></main><footer id="footer" data-type="color"><div id="footer-wrap"><div class="copyright">&copy;2020 - 2022 By Zhu Haizhen</div><div class="framework-info"><span>驱动 </span><a href="https://hexo.io" target="_blank" rel="noopener"><span>Hexo</span></a><span class="footer-separator">|</span><span>主题 </span><a href="https://github.com/jerryc127/hexo-theme-butterfly" target="_blank" rel="noopener"><span>Butterfly</span></a></div><div class="footer_custom_text">君子不器</div></div></footer></div><section class="rightside" id="rightside"><div id="rightside-config-hide"><button id="readmode" title="阅读模式"><i class="fas fa-book-open"></i></button><button id="font_plus" title="放大字体"><i class="fas fa-plus"></i></button><button id="font_minus" title="缩小字体"><i class="fas fa-minus"></i></button><button id="darkmode" title="浅色和深色模式转换"><i class="fas fa-adjust"></i></button></div><div id="rightside-config-show"><button id="rightside_config" title="设置"><i class="fas fa-cog"></i></button><a id="to_comment" href="#post-comment" title="直达评论"><i class="scroll_to_comment fas fa-comments"></i></a><button class="close" id="mobile-toc-button" title="目录"><i class="fas fa-list-ul"></i></button><button id="go-up" title="回到顶部"><i class="fas fa-arrow-up"></i></button></div></section><div class="search-dialog" id="algolia-search"><div class="search-dialog__title" id="algolia-search-title">Algolia</div><div id="algolia-input-panel"><div id="algolia-search-input"></div></div><hr/><div id="algolia-search-results"><div id="algolia-hits"></div><div id="algolia-pagination"></div><div id="algolia-stats"></div></div><span class="search-close-button"><i class="fas fa-times"></i></span></div><div class="search-mask"></div><script src="https://cdn.jsdelivr.net/npm/jquery@latest/dist/jquery.min.js"></script><script src="/js/utils.js"></script><script src="/js/main.js"></script><script src="https://cdn.jsdelivr.net/npm/@fancyapps/fancybox@latest/dist/jquery.fancybox.min.js"></script><script type="text/x-mathjax-config">MathJax.Hub.Config({
  tex2jax: {
    inlineMath: [ ['$','$'], ["\\(","\\)"]  ],
    processEscapes: true,
    skipTags: ['script', 'noscript', 'style', 'textarea', 'pre', 'code']
  },
  CommonHTML: {
    linebreaks: { automatic: true, width: "90% container" }
  },
  "HTML-CSS": { 
    linebreaks: { automatic: true, width: "90% container" }
  },
  "SVG": { 
    linebreaks: { automatic: true, width: "90% container" }
  }
});
</script><script type="text/x-mathjax-config">MathJax.Hub.Queue(function() {
  var all = MathJax.Hub.getAllJax(), i;
  for (i=0; i < all.length; i += 1) {
    all[i].SourceElement().parentNode.className += ' has-jax';
  }
});
</script><script src="https://cdn.jsdelivr.net/npm/mathjax/MathJax.js?config=TeX-AMS-MML_HTMLorMML"></script><script src="https://cdn.jsdelivr.net/npm/node-snackbar/dist/snackbar.min.js"></script><script id="ribbon_piao" mobile="false" src="/js/third-party/piao.js"></script><script src="/js/third-party/activate-power-mode.js"></script><script>POWERMODE.colorful = true;
POWERMODE.shake = true;
document.body.addEventListener('input', POWERMODE);
</script><script async src="//busuanzi.ibruce.info/busuanzi/2.3/busuanzi.pure.mini.js"></script><script src="https://cdn.jsdelivr.net/npm/instant.page/instantpage.min.js" type="module" defer></script><script src="https://cdn.jsdelivr.net/npm/vanilla-lazyload/dist/lazyload.iife.min.js" async></script><script src="/js/third-party/click_heart.js"></script><script src="/js/search/algolia.js"></script><script>var endLoading = function () {
  document.body.style.overflow = 'auto';
  document.getElementById('loading-box').classList.add("loaded")
}
window.addEventListener('load',endLoading)</script><script src="/live2dw/lib/L2Dwidget.min.js?094cbace49a39548bed64abff5988b05"></script><script>L2Dwidget.init({"model":{"jsonPath":"/live2dw/assets/hijiki.model.json"},"display":{"position":"left","width":150,"height":300},"mobile":{"show":false},"log":false,"pluginJsPath":"lib/","pluginModelPath":"assets/","pluginRootPath":"live2dw/","tagMode":false});</script></body></html>